### PR TITLE
Added a GatsbyV2 starter

### PR DESCRIPTION
**Pull request by DanShai**
Starter for Machine learning and scientific writing

### CODE REVIEW

**Comment by DanShai** [comment]
Any volunteer :)

File: docs/starters.yml [hunk 1]
```diff
@@ -1231,3 +1231,21 @@
     - Bootstrap v4 support
     - Css Modules support
     - ESLint, Prettier
+- url: https://danshai.github.io/gatsbyv2-scientific-blog-machine-learning/
+  repo: https://github.com/DanShai/gatsbyv2-scientific-blog-machine-learning
+  description: Machine learning ready and scientific blog starter
+  tags:
+    - GatsbyV2
```
Comment (LekoArts) [inline docs/starters.yml]: You can remove this tag

File: docs/starters.yml [hunk 2]
```diff
@@ -1231,3 +1231,21 @@
...
+    - Blog
+    - Katex
+    - Tensorflow
+    - Csv
```
Comment (LekoArts) [inline docs/starters.yml]: I don't see you using CSV?

Comment (DanShai) [reply docs/starters.yml]: Oh thanx ! I forgot to add it in the last commit..! I will add it shortly

Comment (DanShai) [reply docs/starters.yml]: Done ! csv added , its in here :  https://danshai.github.io/gatsbyv2-scientific-blog-machine-learning/projects/premierelg

File: docs/starters.yml [hunk 3]
```diff
@@ -1231,3 +1231,20 @@
...
     - Css Modules support
     - ESLint, Prettier
+- url: https://danshai.github.io/gatsbyv2-scientific-blog-machine-learning/
+  repo: https://github.com/DanShai/gatsbyv2-scientific-blog-machine-learning
+  description: Machine learning ready and scientific blog starter
+  tags:
+    - Blog
+    - Katex
+    - Tensorflow
+    - Csv
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - CSV
```

File: docs/starters.yml [hunk 4]
```diff
@@ -1231,3 +1231,20 @@
...
+    - Graph mermaid
+    - Linting
+    - Chartjs
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - Chart.js
```

File: docs/starters.yml [hunk 5]
```diff
@@ -1231,3 +1231,20 @@
...
     - ESLint, Prettier
+- url: https://danshai.github.io/gatsbyv2-scientific-blog-machine-learning/
+  repo: https://github.com/DanShai/gatsbyv2-scientific-blog-machine-learning
+  description: Machine learning ready and scientific blog starter
+  tags:
+    - Blog
+    - Katex
+    - Tensorflow
+    - Csv
+    - Graph mermaid
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
    - Graph
```

I think that would be better fitting and more general

**Pull request merged**